### DISCUSSION TITLE
Scaling Images in a Bookmark?

### DISCUSSION THREAD

**Discussion by palzkillm** [organicmaps/organicmaps#5079] [Q&A] [answered]
Hello,

I am creating bookmark files with inline images. Those are currently too small (100x100) for my usage, and I would like to scale them larger. My current way is:

`<img src='data:image/png;base64,iVBORw0KGgoAAAANSUh[...]' width='400' height='400'/>`

This works in a normal browser - but not in the application. A larger image does appear larger, but enlarging the actual image files is unfortunately not an option.

Any suggestions in how to get the browser to scale this larger?

Thanks, Marco

**Comment by biodranik** [comment]
Can you provide an example file? And maybe a screenshot of what you have and how you want to see it?

**Comment by palzkillm** [comment]
Sure! Attached, please find MPA_2023-10_Nuuk_20230513.zip with a KML file. 
Inside this KML, there are 12 placemarks, each with an inline image file of 200x200px. 
I'd like to scale those up to 400x400 using "width='400' height='400', so the image is displayed significantly larger on the screen:
[MPA_2023-10_Nuuk_20230513.zip](https://github.com/organicmaps/organicmaps/files/11469505/MPA_2023-10_Nuuk_20230513.zip)

![image0](https://github.com/organicmaps/organicmaps/assets/107746723/d81ea456-a975-4f53-9d46-f8ffb08458ad)

**Reply by biodranik** [reply]
Did you try to use some other inline styling/css tricks? We never experiment with inline images like you're doing :)

**Reply by palzkillm** [reply]
Not so far. My knowledge about HTML and CSS is pretty crappy, so trying out tricks takes me hours to build it. So before running wild and trying all kinds of stuff and potentially wasting three days for nothing, I wanted to quickly check wether this isnt an issue known to the community. :) Thanks!

**Reply by palzkillm** [reply, marked as answer]
So, I did some very small scale experiments with CSS, but did not succeed: I couldn't scale the images up. I'll close this, it's not that important for me. :)
